src/dto/create-student.dto.ts

```diff
@@ -0,0 +1,23 @@
+import {ContractType, TypeWork} from "../student/student.entity";
+
+export class CreateStudentDto {
+    pwdHash: string;
```

Comment:
For a better naming, I suggest to rename this to just password or pwdPlain, because on this stage it's still just a password. It's gonna become hash when we handle it.